gems/pending/disk/modules/AzureBlobDisk.rb

```diff
@@ -72,7 +72,7 @@ def blob_read(start_byte, length)
     ret = @storage_acct.get_blob_raw(@container, @blob, key, options)
 
     content_md5  = ret.headers[:content_md5].unpack("m0").first.unpack("H*").first
-    returned_md5 = Digest::MD5.hexdigest(ret.body)
+    returned_md5 = Rails.application.config.digest_class.hexdigest(ret.body)
```

Comment:
Am I looking at the wrong post here,  https://blogs.msdn.microsoft.com/windowsazurestorage/2011/02/17/windows-azure-blob-md5-overview/:

> Windows Azure Blob service provides mechanisms to ensure data integrity both at the application and transport layers. This post will detail these mechanisms from the service and client perspective. MD5 checking is optional on both PUT and GET operations; however it does provide a convenience facility to ensure data integrity across the network when using HTTP. Additionally since HTTPS provides transport layer security additional MD5 checking is not needed while connecting over HTTPS as it would be redundant.